labelbox/schema/webhook.py

```diff
@@ -13,35 +20,51 @@ class Webhook(DbObject, Updateable):
         created_at (datetime)
         url (str)
         topics (str): LABEL_CREATED, LABEL_UPDATED, LABEL_DELETED
+            REVIEW_CREATED, REVIEW_UPDATED, REVIEW_DELETED
         status (str): ACTIVE, INACTIVE, REVOKED
 
     """
 
-    # Status
-    ACTIVE = "ACTIVE"
-    INACTIVE = "INACTIVE"
-    REVOKED = "REVOKED"
+    class WebhookStatus(Enum):
```

Comment:
if these are gonna be static sub classes I would maybe just name them `Status`

so you have `Webhook.Status`

Reply:
✔️